src/main/java/org/opensearch/jobscheduler/rest/action/RestGetLockAction.java

```diff
@@ -70,7 +69,7 @@ public List<Route> routes() {
     }
 
     @Override
-    protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient client) throws IOException {
+    public RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient client) throws IOException {
```

Comment:
Curious why `public`? Does it matter?
This interface is anyway called internally by BaseRestHandler[1].

[1] https://github.com/opensearch-project/OpenSearch/blob/0eee9a9403e0338efde497da33fda87c39093b2c/server/src/main/java/org/opensearch/rest/BaseRestHandler.java#L103

Reply:
yes, because if it is not public then I cannot call it from Test class

Reply:
you could make it package private and available by tests